corehq/apps/reports/standard/inspect.py

```diff
@@ -91,11 +91,10 @@ def es_query(self):
                                      lt=self.datespan.enddate_adjusted))
                  .filter(self._get_users_filter(mobile_user_and_group_slugs)))
 
-        # filter results by app and xmlns if applicable
-        if FormsByApplicationFilter.has_selections(self.request):
```

Comment:
Also, it looks like this filter [was added](https://github.com/dimagi/commcare-hq/pull/4466) explicitly to address a [specific issue](http://manage.dimagi.com/default.asp?131005). Good idea to test for regression if it's still applicable.